config/cmake/ConfigureChecks.cmake

```diff
@@ -1,6 +1,7 @@
 #
 # Copyright by The HDF Group.
 # All rights reserved.
+# Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I'm not denying the necessary inclusion - but isn't the COPYING file enough so we don't have to list every copyright holder in every file?

Reply:
NVIDIA's legal guidance for contributing to open source packages says to put the copyright notice on all new files and modified ones where there's another copyright.

Reply:
They also prefer to have the license in each file, but said that if the community standard was to point at a common file, that was acceptable to them.

Reply:
IANAL  🤷‍♂️

Reply:
I only brought it up because of past history and people using cut-n-paste and some files didn't have it, just a real mess. At least now we have a consistent look-n-feel. Also, do we have a policy/doc/file that explains how this is handled. Seems like a good time to nail this down.